node/tests/p2p_test.go

```diff
@@ -37,3 +37,28 @@ func TestUseBridgeNodeAsBootstraper(t *testing.T) {
 		assert.True(t, nodes[index].Host.Network().Connectedness(addr.ID) == network.Connected)
 	}
 }
+
+/*
+Test-Case: Add peer to blacklist
+Steps:
+1. Create a Full Node(BN)
+2. Start a FN
+3. Create a Light Node(LN) and add id of FN to black list
+5. Start a LN
+6. Check FN is allowed to connect to LN
+7. Check LN is not allowed to connect to FN
```

Comment:
Can we elaborate better what does these "allowed/not allowed connect" means? I would rather use terminology that you defined for BlackListPeers `dial` instead of connect. 

In addition, what will happen to the light node, if it has added the full node as trusted peer initially and then opted to blacklist it later? 🧐